client/my-sites/pages/page/index.js

```diff
@@ -459,6 +460,13 @@ class Page extends Component {
 					>
 						{ depthIndicator }
 						{ title }
+						{ latestPostsPage && (
+							<InfoPopover position="right">
+								{ translate(
+									'The content of your latest posts page is automatically generated and it cannot be edited.'
```

Comment:
Not a native speaker myself, but removing the `it` might make the text flow a little better